axon-server/administration/reset-event-processor-token.md

```diff
@@ -0,0 +1,146 @@
+# How to reset the Token of an Event Processor
+
+It might be desirable to reset the position an event processor works on.
+This is done by resetting its token.
+
+We document the following ways to reset the token of an event processor:
+- Using Axon Framework
+- Using the REST API
+- Using the Axon Server Connector
+
+A minimal project showing these approaches can be found [here](https://github.com/AxonIQ/code-samples/tree/master/reset-handler).
+
+## General
+Independent of the method chosen, an event processor needs to be stopped before its token can be reset.
+This requires all instances of the event processor (on all nodes) to release their claims on segments.
+Now, when `resetTokens()` is called, the executing instance can temporarily claim all segments, reset the tokens and store the new values.
+After the token has been reset, the event processor can be started again.
+
+Note that older versions of axon framework (pre 4.6) directly acknowledge a received admin instruction, without waiting for its execution.
+This might require manually waiting for an EventProcessor to be fully stopped before the token can be reset.
+A possible approach for this is shown in the last section of this page.
+
+## Using Axon Framework
+Axon Framework exposes all required functionality for a reset in the `StreamingEventProcessor` class. In the following, we provide a sample for its usage. For more details, read the documentation on the framework classes [here](../../axon-framework/events/event-processors/streaming.md#triggering-a-reset).
+Instances of `StreamingEventProcessor` can be obtained by querying the `eventProcessingConfiguration` method of your global configuration.
+```java
+configuration.eventProcessingConfiguration()
```

Comment:
I think it would be good to define what type the `configuration` is here. 
Right now, users may copy this section and don't know where to find the `configuration` to use here.

In training material, I'd even add the import statement for clarity, although that might not be necessary here.